### ISSUE TITLE
IEEE papers are not 'open access' sensu stricto (just free access)

### ISSUE THREAD

**Issue by rossmounce** [ContentMine/getpapers#27]
```
getpapers -q dinosaurs --api ieee --outdir ./dinos
info: Searching using ieee API
info: Found 9 open access results
```

At least one of the returned papers I looked at is freely available but otherwise full copyright of the IEEE. http://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=1707659
Just a minor language tweak but may I suggest 'free access full text' instead of 'open access' to be on the safe/correct side?

**Comment by Blahah** [comment]
getpapers can only report what it is told by the API - in the case of IEEE there is an 'open access' boolean in the api record. Should we describe them all as 'free download'?

**Comment by rossmounce** [comment]
Yes, I'd err on the side of caution and call them all 'free download' or some such for IEEE.
